app/views/users/new.html.haml

```diff
@@ -1,14 +1,36 @@
+= content_for :head do
+  %script(type="text/javascript" src="https://js.stripe.com/v2/")
+  :javascript
+    Stripe.setPublishableKey("#{ENV['stripe_publishable_key']}");
+  =javascript_include_tag 'payments'
+
 %section.register.container
   .row
     .col-sm-10.col-sm-offset-1
-      = bootstrap_form_for @user, layout: :horizontal, label_col: 'col-sm-2', control_col: 'col-sm-6' do |f|
+      = bootstrap_form_for @user, layout: :horizontal, label_col: 'col-sm-2', control_col: 'col-sm-6', html: { id: 'payment-form' } do |f|
         %header
           %h1 Register
-          = hidden_field_tag :token, @invitation.token if @invitation
+          = hidden_field_tag :invitation_token, @invitation.token if @invitation
         %fieldset
           = f.email_field :email, label: 'Email Address'
           = f.password_field :password
           = f.text_field :username, label: 'Full Name'
+        %fieldset.credit_card
+          %span.payment-errors
+          .form-group
```

Comment:
Apparently, I'm form-challenged... I can't make `= text_field_tag` have the same formatting as the input elements above - because it's plain rails, and not `bootstrap_form`...